.circleci/config.yml

```diff
@@ -238,6 +236,18 @@ jobs:
       ISTIO_TAG: master-latest-daily
     <<: *e2e-dind-test
 
+  e2e-dind-ipv6:
+    <<: *dind-defaults
+    environment:
+      <<: *dind-env-ipv6
+      ISTIO_BRANCH: master
+      ISTIO_HUB: tiswanso
+      ISTIO_TAG: ipv6-13985
+      ISTIO_REMOTE: https://github.com/tiswanso/istio.git
```

Comment:
Nit to consider - Using Istio_remote has the potential to be confused with istio-remote.